dbt/tests/data_tests/generic_tests/expect_unimputed_values_are_unchanged.sql

```diff
@@ -0,0 +1,18 @@
+{% test expect_unimputed_values_are_unchanged(
+    model,
+    original,
+    imputed,
+    imputation_code,
+    atol=0.1,
+    rtol=0.001
+) %}
+SELECT * FROM {{ model }}
+WHERE {{ imputation_code }} IS NULL
+AND (
+    (abs({{ original }} - {{ imputed }}) > {{ atol }})
+    OR (abs({{ original }} - {{ imputed }}) / greatest(abs({{ original }}), abs({{ imputed }})) > {{ rtol }})
```

Comment:
One possible improvement could be to include the absolute/relative error in the `SELECT`, so you can quickly see them if you're trying to tune the tolerances. 

```suggestion
SELECT 
  *,
  (abs({{ original }} - {{ imputed }}) as absolute_error,
  (abs({{ original }} - {{ imputed }}) / greatest(abs({{ original }}) as relative_error
FROM {{ model }}
WHERE {{ imputation_code }} IS NULL
AND ((absolute_error > {{ atol }}) OR (relative_error > {{ rtol }}))
```